dev/doc/MERGING.md

```diff
@@ -41,6 +41,11 @@ patch.
 A maintainer is expected to be reasonably reactive, but no specific timeframe is
 given for reviewing.
 
+(*) In case a component is touched in a trivial way (adding/removing one file in
+a `Makefile`, etc), or by applying a systematic process (global renaming,
+deprecationg propagation, etc) that has been reviwed globally, the assignee can
+say in a comment he thinks a review is not required and proceed with the merge.
```

Comment:
Use they or he/she instead of he: you don't want to assume the assignee's gender in this documentation.